docs/airgap.md

```diff
@@ -26,4 +28,31 @@ sampleRegistry: gcr.io/google-samples
 ```
 
 The keys in that file are specified in the Kubernetes test framework itself. You
-may provide a subset of those and the defaults will be used for the others.
\ No newline at end of file
+may provide a subset of those and the defaults will be used for the others.
+
+## Other required images
+
+The list of custom registries is consumed by the Kubernetes end-to-end tests, but outside of that there are 2 images you will need to be able to access:
+ - the sonobuoy image
+ - the conformance image
+
+To get those images into your cluster you need to pull/tag/push those images yourself:
+
+```
+PRIVATE_REG=<private registry>
+SONO_VERSION=<version of Sonobuoy you are targeting; e.g. v0.14.0>
+CLUSTER_VERSION=<version of k8s you are targeting; e.g. v1.14.0>
+
+docker pull gcr.io/google-containers/conformance:$CLUSTER_VERSION
+docker pull gcr.io/heptio-images/sonobuoy:$SONO_VERSION
+
+docker tag gcr.io/google-containers/conformance:$CLUSTER_VERSION $PRIVATE_REG/conformance:$CLUSTER_VERSION
+docker tag gcr.io/heptio-images/sonobuoy:$SONO_VERSION $PRIVATE_REG/sonobuoy:$SONO_VERSION
+
+docker push $PRIVATE_REG/conformance:$CLUSTER_VERSION
+docker push $PRIVATE_REG/sonobuoy:$SONO_VERSION
+```
+
+If you want to run the systemd_logs plugin you'll need to pull/tag/push it as well. In addition, you'll have to manually specify the image you want to use via `sonobuoy gen` -> `kubectl apply` since that image is not overridable on the CLI.
```

Comment:
Maybe specify the value to replace here for the plugin so folks know what to look for? (e.g. `gcr.io/heptio-images/sonobuoy-plugin-systemd-logs:latest`)

Also might be nice to next add docs section on how to edit plugins (add / remove) and can link to that from here.